pyproject.toml

```diff
@@ -75,9 +75,16 @@ lab = [
     "notebook >=7",
 ]
 
+[project.urls]
+Documentation = "https://jupyter-server-proxy.readthedocs.io"
+Source = "https://github.com/jupyterhub/jupyter-server-proxy"
+Tracker = "https://github.com/jupyterhub/jupyter-server-proxy/issues"
+
+
+# hatch ref: https://hatch.pypa.io/latest/
+#
 [tool.hatch.version]
-source = "nodejs"
-path = "labextension/package.json"
+path = "jupyter_server_proxy/_version.py"
```

Comment:
if using `importlib.metdata`, would be able to drop this, as the `pyproject.toml` would be the single-source-of-truth for the python distribution.